src/main/java/com/jcabi/github/mock/MkRepos.java

```diff
@@ -90,8 +90,13 @@ public Github github() {
     public Repo create(
         final RepoCreate settings
     ) throws IOException {
+        String user = this.self;
```

Comment:
@maviteixeira looks good; can you just rename this variable to ``owner``? As in, the ``owner`` is either ``this.self`` (the authenticated user) or an organization.

Reply:
Sure, no problem.